packages/vite/package.json

```diff
@@ -63,12 +63,9 @@
     "dev": "rimraf dist && pnpm run build-bundle -w",
     "build": "rimraf dist && run-s build-bundle build-types",
     "build-bundle": "rollup --config rollup.config.ts --configPlugin typescript",
-    "build-types": "run-s build-types-temp build-types-pre-patch build-types-roll build-types-post-patch build-types-check",
+    "build-types": "run-s build-types-temp build-types-roll",
     "build-types-temp": "tsc --emitDeclarationOnly --outDir temp/node -p src/node",
-    "build-types-pre-patch": "tsx scripts/prePatchTypes.ts",
-    "build-types-roll": "tsx scripts/api-extractor.ts run && rimraf temp",
-    "build-types-post-patch": "tsx scripts/postPatchTypes.ts",
-    "build-types-check": "tsx scripts/checkBuiltTypes.ts && tsc --project tsconfig.check.json",
```

Comment:
We still need to run `tsc --project tsconfig.check.json` to make sure the types work with `"exactOptionalPropertyTypes": true`.

Reply:
Ah thanks. I missed that additional command there.